LICENSE

```diff
@@ -1,21 +1,22 @@
 MIT License
 
-Copyright (c) 2021 MrWardKKHS
+Copyright (c) 2021 Jbeale
```

Comment:
This change is unacceptable. Please fix

Reply:
> 
[https://www.youtube.com/watch?v=HIcSWuKMwOw](url)

Reply:
This is a rick roll. Please correct.